azure-pipelines.yml

```diff
@@ -1,13 +1,15 @@
-# Starter pipeline
-# Start with a minimal pipeline that you can customize to build and deploy your code.
-# Add steps that build, run tests, deploy, and more:
-# https://aka.ms/yaml
+name: $(Build.SourceBranchName)_$(Build.Reason)_$(devops_buildNumber)
 
 pool:
-  vmImage: 'Ubuntu 16.04'
+  vmImage: 'vs2017-win2016'
```

Comment:
is there a vs2019 image?

Reply:
not sure, this i what we had of kafka so i used the same